be/src/common/status.h

```diff
@@ -291,125 +279,125 @@ class Status {
     }
 
     template <typename... Args>
-    static Status InternalError(const std::string& fmt, Args&&... args) {
+    static Status InternalError(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::INTERNAL_ERROR, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status RuntimeError(const std::string& fmt, Args&&... args) {
+    static Status RuntimeError(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::RUNTIME_ERROR, fmt, std::forward<Args>(args)...);
     }
     template <typename... Args>
-    static Status Cancelled(const std::string& fmt, Args&&... args) {
+    static Status Cancelled(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::CANCELLED, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status MemoryLimitExceeded(const std::string& fmt, Args&&... args) {
+    static Status MemoryLimitExceeded(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::MEM_LIMIT_EXCEEDED, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status ThriftRpcError(const std::string& fmt, Args&&... args) {
+    static Status RpcError(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::THRIFT_RPC_ERROR, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status TimedOut(const std::string& fmt, Args&&... args) {
+    static Status TimedOut(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::TIMEOUT, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status TooManyTasks(const std::string& fmt, Args&&... args) {
+    static Status TooManyTasks(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::TOO_MANY_TASKS, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status ServiceUnavailable(const std::string& fmt, Args&&... args) {
+    static Status ServiceUnavailable(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::SERVICE_UNAVAILABLE, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status Uninitialized(const std::string& fmt, Args&&... args) {
+    static Status Uninitialized(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::UNINITIALIZED, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status Aborted(const std::string& fmt, Args&&... args) {
+    static Status Aborted(std::string_view fmt, Args&&... args) {
         return ErrorFmt(TStatusCode::ABORTED, fmt, std::forward<Args>(args)...);
     }
 
     template <typename... Args>
-    static Status DataQualityError(const std::string& fmt, Args&&... args) {
+    static Status DataQualityError(std::string_view fmt, Args&&... args) {
```

Comment:
I find only OLAPInternalError has error stack, static Status DataQualityError(std::string_view fmt, Args&&... args) these method does not generate error stack.